test/parallel/test-buffer-safe-unsafe.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+require('../common');
+const SlowBuffer = require('buffer').SlowBuffer;
+const assert = require('assert');
+
+const safe = Buffer.zalloc(10);
+const safeslow = SlowBuffer.zalloc(10);
+
+function isZeroFilled(buf) {
+  for (let n = 0; n < buf.length; n++)
+    if (buf[n] > 0) return false;
+  return true;
+}
```

Comment:
while that may increase the likelihood of hitting possible failure, won't rule it out. guess we can just do a best possible approach.